deploy/ods-pipeline/values.kind.yaml

```diff
@@ -49,43 +49,12 @@ setup:
       - 'Failed'
     # Template to be processed and accepted by the configured webhook in use
     # Below example might work for Microsoft Teams
-    requestTemplate: |-
```

Comment:
Why is this not needed anymore?

Reply:
* this values file is only used in KinD clusters
* the whole feature is disabled (see a bit further up in the file) so it doesn't matter anyway what the value for `requestTemplate` is